lightyear/src/shared/config.rs

```diff
@@ -17,14 +17,32 @@ pub struct SharedConfig {
     pub server_send_interval: Duration,
     /// configuration for the [`FixedUpdate`](bevy::prelude::FixedUpdate) schedule
     pub tick: TickConfig,
-    /// do we run in unified mode? (client and server in the same process)
-    pub unified: bool,
+    pub mode: Mode,
+}
+
+#[derive(Default, Clone, Copy, Debug, PartialEq, Eq)]
+pub enum Mode {
+    #[default]
+    /// Run the client and server in two different apps
+    Separate,
+    /// Run the client and server in the same bevy app/world
+    /// NOTE: this is complicated to get right, and is not recommended, as it is hard to
+    /// distinguish between client entities and server entities
+    Unified,
```

Comment:
Should this be removed given that it's unworkable?

Reply:
Yes, I think i'll remove it